Casks/lilypond-unofficial.rb

```diff
@@ -0,0 +1,45 @@
+cask 'lilypond-unofficial' do
+  version '2.20.0'
+  sha256 '655bb00970279ae72d3c7dfe19186db9025f0cb5eae7e3c4485c1b3b1116bc69'
+
+  # bintray.com/marnen/lilypond-darwin-64/download_file?file_path=lilypond- was verified as official when first introduced to the cask
+  url "https://bintray.com/marnen/lilypond-darwin-64/download_file?file_path=lilypond-#{version}.build20200311175017-darwin-64.tar.gz"
+  appcast 'https://bintray.com/marnen/lilypond-darwin-64/lilypond-2.20.0#release'
```

Comment:
On line 2, you declare `version '2.20.0'`. However, I would hazard to guess (and @marnen can correct me if I'm wrong) that the build number (currently `20200311175017`) would normally change more often than the patch number. When I tried my hand at creating a cask for this, I declared `version '2.20.0.build20200311175017'`, but then I found [a way that I think is even better](https://github.com/Homebrew/homebrew-cask/blob/6b275048d64c533e618eaa4a8a07f157b1cc534c/Casks/preform.rb) because it allows the part of the version number before the build number to be used in one more place.

Reply:
@fanninpm Thank you very much for this tip. I correct right away.

However, I did not understand the `EL_NOMBRE_DE_SU_RAMA`.

Reply:
It seems from my latest push that you got the gist of what I meant by `LE_NOME_DE_VOTER_BRANCH`, but it's currently late at night where I am.

You may want to add the version number to the `appcast` string (but I would personally **hold off** from doing that until the maintainer comes by to comment that this is the correct way to go).

Reply:
@fanninpm:
> However, I would hazard to guess (and @marnen can correct me if I'm wrong) that the build number (currently 20200311175017) would normally change more often than the patch number.

That's generally correct, although at this point, they may change at the same rate. I originally had the build number in there because I had to develop a new build process for 64-bit Mac builds, and since I might build the same source code with several different attempts at a Makefile that worked, it made sense to have a build number so I could track which version of the Makefile had been used for which build.

Now that the Makefile more or less works, the build number won't change that often (probably just when I actually build from a new version of the source, or when the Makefile requires major changes).